CMSgov/design-system
# [WNMGDS-2912] Updates versions script and adds latest patch and release versions

## Summary

Right now, multiple beta versions are deployed to https://design.cms.gov/cdn when there should only be one (if any). This script updates the "bump version" logic in the release script to only show the most recent beta version before a new major version has been deployed.

Separately, this updates the versions.json to include the latest version and patch version (9.0.3) since those were not included in prior releases as a result of sorting out the release script.

## How to test

1. In the [release.ts](https://github.com/CMSgov/design-system/blob/main/scripts/release.ts#L100-L137) script, comment out Lines 100-137 and [Lines 222-224](https://github.com/CMSgov/design-system/blob/main/scripts/release.ts#L222-L224).
2. Run `yarn release` and select `prerelease`. The `versions.json` changes should add the current beta version (e.g. `11.0.0-beta.2` in favor of `11.0.0-beta.1` and `11.0.0-beta.0`)

## Checklist
